docs/ELRS_RECEIVERS.md

```diff
@@ -14,27 +14,29 @@ As of April 2024, ELRS receivers with an ESP8285 MCU are supported by mLRS. The
     </tr>
     <tr>
       <td>Generic 900</td>
-      <td>17 dBm (50 mw)</td>
+      <td>17 dBm (50 mW)</td>
     </tr>
     <tr>
       <td>Generic 900 PA</td>
-      <td>27 dBm (500 mw)</td>
+      <td>27 dBm (500 mW)</td>
     </tr>
     <tr>
       <td>Generic 2400</td>
-      <td>12.5 dBm (18 mw)</td>
+      <td>12.5 dBm (18 mW)</td>
     </tr>
       <td>Generic 2400 PA</td>
-      <td>20 dBm (100 mw)</td>
+      <td>20 dBm (100 mW)</td>
+    </tr>
+      </tr>
+      <td>Generic 2400 True Diversity PA</td>
```

Comment:
I wonder if TA PA would be better, at least this would be how to find the firmware (and also in VSCode)

maybe further colum, with it's options in plain, like PA + LNA, true diversity

am just thinking

Reply:
Okay, maybe an attempt.